### PR TITLE
Add WikiWinInstall

### PR DESCRIPTION
**Pull request by OlivierJB**
Powershell/shell  script allowing for one click install and build of Ardupilot wiki on windows local machine under cygwin.

Download and right-click "Run with Powershell".

Similar in spirit to:
 https://github.com/ArduPilot/ardupilot/blob/master/Tools/environment_install/install-prereqs-windows.ps1

Tested on windows 10, incompatible with win8.1. and 7
Note: Can be run on top of existing cygwin install (eg with previous cygwin Ardupilot sitl build) If ~/ardupilot_wiki directory alreadyexists it will not be overwritten.

### CODE REVIEW

File: Tools/environment_install/APWikiWinInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+spinner & >> $logfile 2>&1
+SPID=$! && disown
+
+printf '\nInstalling Ardupilot Wiki and build environment within Cygwin\n'
+printf '[Open 2nd Cygwin terminal and use tail -f /tmp/*.log for verbose progress output]\n\n'
+printf 'This may take several minutes ...\n\n'
+
+# Get pip
+ProgressBar 3 "Getting pip           "
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py > $logfile 2>&1
```
Comment (khancyr) [inline Tools/environment_install/APWikiWinInstall.ps1]: I am a little unconfortable with those instruction as they are duplicate from the Sphinxsetup script but won't be updated if we make changes on wiki side. Can't we just use the script ?

**Comment by stephendade** [comment]
And as @khancyr mentioned, make use of the existing sphinx setup file in the wiki repo if practical.

**Comment by WickedShell** [comment]
I haven't reviewed this in detail, but if its the setup the wiki script why not have it in the wiki repo, instead of the ArduPilot one?

File: Tools/environment_install/APWikiWinInstall.ps1 (hunk 1 above)
Comment (OlivierJB) [reply Tools/environment_install/APWikiWinInstall.ps1]: I agree that's a potential  issue, although the sphinxsetup script is small and simple. The problem with  using it t is that it has apt-get. Not gonna work well on cygwin ...  ;)

**Comment by OlivierJB** [comment]
@stephendade thanks. yeah, let me look into it.
@WickedShell That would make sense. I think I just followed similar script location https://github.com/ArduPilot/ardupilot/blob/master/Tools/environment_install/install-prereqs-windows.ps1

**Comment by WickedShell** [comment]
@OlivierJB Yeah, I *think* (could be wrong) that script is about installing the tools needed to build ArduPilot on windows though, not anything with the wiki. (Still a valid template to use, just not sure the new script is being aimed at the right repo)

**Comment by khancyr** [comment]
As said previously, I prefer to not merge this as it belong to wiki repo. 
Moreover, it brings duplicated instruction from the wiki installation steps that will make maintenance harder.

@Hwurzburg Would you want some work to port those script into the wiki repo or we close this definitely ?

**Comment by OlivierJB** [comment]
Yeah, should have put it in wiki repo.
Closing for now, can always relocate to wiki if there is some interest for it. I do think it's a nice addition for cygwin env. that greatly simplify set-up,  but there doesn't seem to be much AP cygwin users with the need for it.

**Comment by Hwurzburg** [comment]
at least in Win8/10 using WSL is easy, built-in, and works fine....using the Linux wiki builder instructions and setup...no need for cygwin

**Pull request closed without merging**